frontend/wallet/src/render/components/TransactionsView.re

```diff
@@ -92,14 +110,35 @@ let extractTransactions: Js.t('a) => array(TransactionCell.Transaction.t) =
 
 [@react.component]
 let make = () => {
-  let transactionQuery = Transactions.make(~publicKey="123", ());
+  let activeWallet = Hooks.useActiveWallet();
+  let activeWalletKey =
+    Option.map(~f=PublicKey.toString, activeWallet)
+    |> Option.withDefault(~default="");
+  let transactionQuery = Transactions.make(~publicKey="123", ~after="", ());
+  let (isFetchingMore, setFetchingMore) = React.useState(() => false);
+
+  let keyForTransaction = (data, index) =>
+    Option.withDefault(~default="", data##blocks##pageInfo##lastCursor)
+    ++ string_of_int(index);
+
+  let updateQuery: ReasonApolloQuery.updateQueryT = [%bs.raw
```

Comment:
This is definitely the most gnarly bit, reason-apollo doesn't provide valid bindings for the update function so we have to just bs raw it. :( ref. https://github.com/apollographql/reason-apollo/blob/c027522cf9e3aa30bfcc79f36dc8271b6cc327d9/examples/swapi/src/Persons.re#L44